src/components/CreateOrUpdateRequest.svelte

```diff
@@ -88,8 +81,14 @@ async function cancelRequest() {
   cancelled()
 }
 
+function onEventChange(e) {
+  request.event_id = e.detail
+  request.destination = $events.find(({ id }) => id === e.detail).location
```

Comment:
not sure, but are the () needed around the object parameter?

Reply:
Good question. I just tried without...no good